.github/workflows/toolchain_build.yml

```diff
@@ -0,0 +1,106 @@
+name: Build toolchains
+
+on:
+  push:
+    branches: ['master']
+    tags: ['*']
+  pull_request:
+    branches: ['master']
+
+env:
+  ARTIFACT_STAGING_DIR: artifact
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-20.04
+    timeout-minutes: 360
+
+    strategy:
+      matrix:
+        include:
+          - name: rv32imcb
+            target: riscv32-unknown-elf
+            output_dir: /tools/riscv
+            march: rv32imc_zba_zbb_zbc_zbs
+            mabi: ilp32
+            mcmodel: medany
+          - name: rv64imac
+            target: riscv64-unknown-elf
+            output_dir: /tools/riscv
+            march: rv64imac
+            mabi: lp64
+            mcmodel: medany
+#         - name: multilib-baremetal
+#           target: riscv64-unknown-elf
+#           output_dir: /opt/riscv-baremetal-toolchain
+#         - name: multilib-linux
+#           target: riscv64-unknown-linux-gnu
+#           output_dir: /opt/riscv-linux-toolchain
```

Comment:
Are these not working?

Reply:
They worked but require a longer timeout. They were disabled in the Azure pipeline, so I've included them here commented out. I guess we weren't using the?

Reply:
Ok, that makes sense